src/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -29,7 +29,8 @@ export function AggTypesBucketsDateHistogramProvider(timefilter, config, Private
   function setBounds(agg, force) {
     if (agg.buckets._alreadySet && !force) return;
     agg.buckets._alreadySet = true;
-    agg.buckets.setBounds(agg.fieldIsTimeField() && timefilter.getActiveBounds());
+    const timeRange = timefilter.getActiveBounds() || agg.params.timeRange;
```

Comment:
Can you change the order of the `OR` statement so that a provided `timeRange` takes precedent over the` timefilter.getActiveBounds`? @alexfrancoeur commented on [embed visualization in tooltip](https://github.com/elastic/kibana/pull/14102#issuecomment-333592985) and made a good point that if you are looking at an embedded tooltip for a date_histogram - it would be nice to show the embedded tooltip visualization for only the highlighted time range. Switching the order of this `OR` statement will allow this.